grpc.go

```diff
@@ -934,6 +945,38 @@ func (a *agentGRPC) SignalProcess(ctx context.Context, req *pb.SignalProcessRequ
 	return emptyResp, nil
 }
 
+// Check is the container process installed the
+// handler for "term" signal.
+func isSigTermCatched(pid int) bool {
+	var termMask uint64 = 1 << (uint(syscall.SIGTERM) - 1)
+	procFile := fmt.Sprintf("/proc/%d/status", pid)
+	file, err := os.Open(procFile)
+	if err != nil {
+		agentLog.Warnf("Open proc file %s failed", procFile)
```

Comment:
I think this should be an error and `return false`

```go
agentLog.WithField("pid", pid).WithError(err).Error("Could not open status file")
return false
```

Reply:
Yes, otherwise the following code will crash when accessing `file`.

Reply:
please use `WithField`

```go
agentLog.WithField("pid", pid).WithError(err).Error("Could not open status file")
```

Reply:
:smile: